src/capi.cpp

```diff
@@ -1144,6 +1144,22 @@ int rtcGetTrackDirection(int tr, rtcDirection *direction) {
 	});
 }
 
+int rtcSendTrackRequestKeyframe(int tr) {
```

Comment:
I think it could be renamed for consistency with the rest of the API:
```suggestion
int rtcRequestKeyframe(int tr) {
```